phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -208,6 +209,195 @@ protected static void setEveryNthRowWithNull(int nrows, int nthRowNull, Prepared
         }
     }
 
+    @Test
+    public void testUpdatableViewIndex() throws Exception {
```

Comment:
Please move this to IndexToolForNonTxGlobalIndexIT

Reply:
Done